cmd/disk_wipe.go

```diff
@@ -74,14 +76,27 @@ func init() {
 
 			// Pick the most appropriate wipe based on the disk type and/or features supported
 			var wiper actions.DriveWiper
-			// nolint:gocritic // will have more cases soon, remove nolint then
 			switch drive.Protocol {
 			case "nvme":
 				wiper = utils.NewNvmeCmd(verbose)
+			case "sata":
+				// Lets figure out if the drive supports TRIM
+				var trim bool
+				for _, cap := range drive.Capabilities {
+					if strings.HasPrefix(cap.Description, "Data Set Management TRIM supported") {
+						trim = cap.Enabled
+						break
+					}
+				}
+
+				if trim {
+					// Drive supports TRIM, so we use blkdiscard
+					wiper = utils.NewBlkdiscardCmd(verbose)
+				}
 			}
 
 			if wiper == nil {
-				l.Fatal("failed find appropriate wiper drive")
+				l.Fatal("failed find appropriate drive wiper")
```

Comment:
for debugging, in the fatal error - it may be worth including information on the drive protocol and trim support information?

Reply:
drive protocol maybe but nvme's won't support trim maybe I just dump the capabilities?